api/filters/labels/labels.go

```diff
@@ -21,22 +21,15 @@ type Filter struct {
 	// FsSlice identifies the label fields.
 	FsSlice types.FsSlice
 
-	// SetEntryCallback is invoked each time a label is applied
-	// Example use cases:
-	//   - Tracking all paths where labels have been applied
-	SetEntryCallback func(key, value, tag string, node *yaml.RNode)
+	setter filtersutil.TrackableSetter
```

Comment:
nit: suggest changing the name from `setter` to `trackableSetter`. The word "setter" is overloaded in our space so additional context is always helpful.

Reply:
good suggestion - done!